zebra-state/src/lib.rs

```diff
@@ -87,10 +87,12 @@ pub enum Response {
         /// The hash of the block at the tip of the current chain
         hash: BlockHeaderHash,
     },
-    /// The  response to a `GetKnownBlockHashes` request
-    KnownBlockHashes {
-        /// the hashes of all known blocks that were requested
-        hashes: Vec<BlockHeaderHash>,
+    /// The response to a `Contains` request indicating that the given has is in
+    /// the current best chain
+    Contained {
```

Comment:
```suggestion
    Depth {
```
Calling this "depth" lets us signal that the block might not be contained in the main chain in a way that "contained" doesn't.